### PR TITLE
Allow VCA members to remove patients.

### PR DESCRIPTION
**Pull request by cbwebdevelopment**
Details at https://trello.com/c/LIrmtf4i

### CODE REVIEW

**Comment by cbwebdevelopment** [comment]
[Remove clinic account from care team](https://trello.com/c/LIrmtf4i/188-remove-clinic-account-from-care-team)

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "react-redux": "4.4.5",
     "react-router": "2.7.0",
     "react-router-redux": "3.0.0",
+    "react-window-resize-listener": "1.1.0",
```
Comment (cbwebdevelopment) [inline package.json]: I agree.  This repo looks pretty stale.  I've changed over to the suggested forked repo.

File: app/components/peopletable/peopletable.less [hunk 2]
```diff
@@ -15,34 +15,43 @@
...
-    cursor: pointer;
-    color: @black-color;
-    font-size: @font-size-base;
-    text-decoration: none;
-    &:hover,
-    &:active {
-      color: @link-alternative-hover-color;
-    }
-  }
+.peopletable-names-toggle-wrapper {
```
Comment (cbwebdevelopment) [inline app/components/peopletable/peopletable.less]: Indeed :) Good catch!

**Comment by krystophv** [comment]
This looks good to me. Be sure to schedule a design review with the relevant parties.

**Pull request merged**